src/app/[locale]/page.tsx

```diff
@@ -19,11 +19,12 @@ export default async function Home({searchParams}: Props) {
     sortOrder: searchOptions.sortOrder,
   });
 
-  let hasError, searchResult;
+  let hasError, searchResult, errorMessage;
   try {
     searchResult = await datasetFetcher.search(searchOptions);
   } catch (error) {
     hasError = true;
+    errorMessage = error;
```

Comment:
The `error` is of type `Error`. Do you want to get just the error message? If so, use `errorMessage = error.message`? Or do you want to have the full error object? In that case you could do a `JSON.stringify(error, null, 2)}`?

Reply:
This branch is not meant to be merged but is just for debugging. So I did not really think about good code.